manager/manager/utils.py

```diff
@@ -733,6 +736,25 @@ def get_obsday_from_tai(tai):
     return observing_day
 
 
+def get_obsday_to_tai(obsday):
```

Comment:
I think it would make sense to call this `get_day_obs_to_tai`, rather than obsday, unless these are separate meanings. But based on the use above I feel these might be the same?

Reply:
So I use `obsday` here in the code to refer to the `YYYYMMDD` format of the day of observation. There are -now- three methods in the `utils.py` which refer to utils to parse these values:

- get_obsday_from_tai -> convert a tai date to `YYYYMMDD` string format.
- get_obsday_to_tai -> convert an obsday in `YYYYMMDD` int or string format to a tai date.
- get_obsday_iso -> convert an obsday in `ỲYYYMMDD` int or string format to ISO-truncated format: YYYY-MM-DD.